weatherApp/src/screens/CitiesStackScreen/index.tsx

```diff
@@ -0,0 +1,31 @@
+import {createNativeStackNavigator} from '@react-navigation/native-stack';
+import React from 'react';
+import AddCityScreen from '../AddCityScreen';
+import CitiesScreen from '../CitiesScreen';
+import {observer} from 'mobx-react';
+
+export type RootStackParamList = {
+  CitiesScreen: undefined;
+  AddCityScreen: undefined;
+};
+
+const CitiesStackScreen = () => {
```

Comment:
it's just a stack. You can name it CitiesStack and move it to the navigation folder.